Makefile

```diff
@@ -427,3 +427,6 @@ check_abci_specs:
 	python -m autonomy.cli analyse abci generate-app-specs packages.valory.skills.transaction_settlement_abci.rounds.TransactionSubmissionAbciApp packages/valory/skills/transaction_settlement_abci/fsm_specification.yaml || (echo "Failed to check transaction_settlement_abci consistency" && exit 1)
 	echo "Successfully validated abcis!"
 
+
+release-images:
```

Comment:
calls skaffold build with the release profile (Which we set the agent-hash of the hello-world agent) 

Then calls it again with  the env var set for the tag

Reply:
LGTM